examples/minicpm/chat.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import ast
+import os
+import sys
+
+__dir__ = os.path.dirname(os.path.abspath(__file__))
```

Comment:
it's better to avoid setting abs path. 
This can be removed If `pip install -e .`